### ISSUE TITLE
policy_web, policy_infra, policy_tunnel - cannot update specs

### ISSUE THREAD

**Issue by tdesikan** [banyansecurity/terraform-provider-banyan#76]
when you try to update a policy, you get an error:

```
banyan_policy_tunnel.datatcenter_l4_controls: Modifying... [id=6f6d2364-5b67-441e-888c-795e2409a539]
╷
│ Error: couldn't create new tunnel policy: A existing policy was found with name Datacenter w L4 Controls (id=6f6d2364-5b67-441e-888c-795e2409a539)
│
│   with banyan_policy_tunnel.datatcenter_l4_controls,
│   on policies.tf line 27, in resource "banyan_policy_tunnel" "datatcenter_l4_controls":
│   27: resource "banyan_policy_tunnel" "datatcenter_l4_controls" {
│
╵
```

In the code, we're doing a `resourcePolicyTunnelCreate` even for an update. the API doesn't like it.

**Comment by tdesikan** [comment]
doesn't look like an API issue. looks like the recently added check is preventing updates.

```
	existing, err := p.GetName(policy.Metadata.Name)
	if existing.Name == policy.Metadata.Name {
		err = fmt.Errorf("A existing policy was found with name %s (id=%s)", existing.Name, existing.ID)
		return
	}
```

**Comment by tdesikan** [comment]
closed by #73
